pypechain/utilities/abi.py

```diff
@@ -557,6 +556,28 @@ def get_input_names_and_values(function: ABIFunction) -> list[str]:
     return _get_names_and_values(function, "inputs")
 
 
+def get_output_types(function: ABIFunction) -> list[str]:
+    """Returns function output type strings for jinja templating.
+
+    i.e. for the solidity function signature: function doThing(address who, uint256 amount, bool
+    flag, bytes extraData)
+
+    the following list would be returned: ['who: str', 'amount: int', 'flag: bool', 'extraData:
+    bytes']
+
+    Arguments
+    ---------
+    function : ABIFunction
+        A web3 dict of an ABI function description.
+
+    Returns
+    -------
+    list[str]
+        A list of function python values, i.e. ['str', 'bool']
```

Comment:
This doesn't match the example at the top of the docstring. wouldn't it be ['var: type']?

Reply:
fixed the comment above which was wrong.